### PR TITLE
Feat/player default audio track

### PR DESCRIPTION
**Pull request by tymmesyde**
Add audio track setting to Settings
Select default audio track on Player

### CODE REVIEW

File: src/routes/Player/Player.js [hunk 1]
```diff
@@ -313,6 +315,17 @@ const Player = ({ urlParams, queryParams }) => {
             pausedChanged(videoState.paused);
         }
     }, [videoState.paused]);
+    React.useEffect(() => {
+        if (defaultAudioTrackSelected.current === false) {
```
Comment (nklhtv) [inline src/routes/Player/Player.js]: `if (!defaultAudioTrackSelected.current) {`
looks better

File: src/routes/Player/Player.js [hunk 2]
```diff
@@ -40,6 +41,7 @@ const Player = ({ urlParams, queryParams }) => {
     const [subtitlesMenuOpen, , closeSubtitlesMenu, toggleSubtitlesMenu] = useBinaryState(false);
     const [infoMenuOpen, , closeInfoMenu, toggleInfoMenu] = useBinaryState(false);
     const [videosMenuOpen, , closeVideosMenu, toggleVideosMenu] = useBinaryState(false);
+    const defaultAudioTrackSelected = React.useRef(false);
```
Comment (nklhtv) [inline src/routes/Player/Player.js:46]: reset defaultAudioTrackSelected to false when videoState.stream changes

File: src/routes/Player/Player.js [hunk 3]
```diff
@@ -313,6 +315,17 @@ const Player = ({ urlParams, queryParams }) => {
...
+        if (defaultAudioTrackSelected.current === false) {
+            const findTrackByLang = (tracks, lang) => tracks.find((track) => track.lang === lang || langs.where('1', track.lang)?.[2] === lang);
+            const audioTrack = findTrackByLang(videoState.audioTracks, settings.audioLanguage);
+
+            if (audioTrack && audioTrack.id) {
+                onAudioTrackSelected(audioTrack.id);
+                defaultAudioTrackSelected.current = true;
+            }
+        }
+    }, [videoState.audioTracks, settings.audioLanguage]);
```
Comment (nklhtv) [inline src/routes/Player/Player.js]: Should we trigger this effect when `settings.audioLanguage` change

**Review by nklhtv — CHANGES_REQUESTED** [review]

File: src/routes/Player/Player.js (hunk 3 above)
Comment (tymmesyde) [reply src/routes/Player/Player.js]: Maybe that's not necessary yes

**Review by nklhtv — APPROVED** [review]

**Pull request merged**